packages/nextjs/components/NetworksDropdown.tsx

```diff
@@ -70,6 +80,7 @@ export const NetworksDropdown = ({ onChange }: { onChange: (options: any) => any
     }
   }, []);
 
+  if (!mounted) return null;
```

Comment:
Yeah there is no other way round to this :( 


https://github.com/BuidlGuidl/abi.ninja/assets/80153681/293cb9c6-66fa-46d9-bdad-64037e43cd0e

Nipick: Maybe we keep an dummy div with same width / height maybe even [skeleton](https://daisyui.com/components/skeleton/) to workaround this layout shift as suggested here : 
![Screenshot 2024-05-07 at 9 01 05 PM](https://github.com/BuidlGuidl/abi.ninja/assets/80153681/ba06ed49-2675-4b84-8cf2-85e0b9824d7f)

https://github.com/pacocoursey/next-themes?tab=readme-ov-file#avoid-hydration-mismatch